packages/lit/src/index.max.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import './polyfill-support.js';
```

Comment:
I think this file should have a console.warn() about not using it in production and it being much larger than using the files from npm.

Reply:
Ok, I think I'll also add a flag to disable it though.

Reply:
Added.